docs/game_sdk/Applications.md

```diff
@@ -30,6 +30,38 @@ This manager also includes a couple useful helper functions, like getting the lo
 | entitlements   | list of partial [Entitlements](#DOCS_GAME_SDK_STORE/data-models-entitlement-struct) structs that contain just the SKU id | the list of the user's entitlements for this application |
 | timestamp      | string                                                                                                                   | the ISO 8601 timestamp for the ticket                    |
 
+###### Locales
```

Comment:
This list is actually already available here: https://discordapp.com/developers/docs/dispatch/field-values#predefined-field-values-accepted-locales

So I don't think we need to double-add

Reply:
I'll make the adjustment